dockerfiles/Dockerfile-centos7

```diff
@@ -17,11 +17,10 @@
 # under the License.
 #
 
-# Downloads, builds and installs apache proton and qpid dispatch on Centos and starts the dispatch router
```

Comment:
There is a file called Dockerfile in the qpid-dispatch base folder. That file could probably be moved to use rhel 8 instead of rhel 7

Reply:
Please commit after @kgiusti has had a chance to look at this. thx

Reply:
CentOS 8 has only a year to live. It looks that it would make sense to use RHEL 8 UBI image, as per https://wiki.centos.org/FAQ/CentOSStream#What_artifacts_are_built.3F.

![image](https://user-images.githubusercontent.com/442720/107942660-da632500-6f8b-11eb-8cc1-dc45365e44a6.png)

Reply:
Using RHEL 8 UBI makes sense to me

Reply:
++, it's not related to Python 2.6 in any way, though. I won't be addressing it in this PR.

Reply:
Agreed. But should dockerfiles/Dockerfile-centos6  be simply removed  instead of renaming it to Dockerfile-centos7 ?

Reply:
That would make sense. Actually, I did not notice there is a top level Dockerfile, before you pointed that out. I somehow assumed all dockerfiles are in the dedicated directory.

Afaik there should remain a centos 7 dockerfile, due to compatibility testing. It is the oldest centos that Dispatch will support. If the top level Dockerfile goes to RHEL 8 UBI, there should be a CentOS 7 docker somewhere too.

Reply:
So, I would leave the PR as it is, and then update the top level dockerfile to RHEL 8 UBI / CentOS stream, when that becomes available.

Reply:
Sounds good. Can you please raise a JIRA so the main Dockerfile can be moved to RHEL 8 UBI at some point. The new JIRA is just keep track of the work that needs to be done.

Reply:
https://issues.apache.org/jira/browse/DISPATCH-1966